package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent",
   "description": "Streaming torrent client",
-  "version": "0.96.5",
+  "version": "0.97.0",
```

Comment:
In the future, please don't change the `package.json` version in PRs. That's best done with the `npm version` command (i.e. `npm version patch`, `npm version minor`, `npm version major`) which creates a standalone commit where the version is changed. This makes it easier to spot releases in the changelog. It also takes care of creating a git tag on that same commit.